<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyVocabulary</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_box">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Type here to search online...</string>
    </property>
   </widget>
   <widget class="QLabel" name="word_count">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>630</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="word_table">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>500</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>English</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Turkish</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>600</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selected Words</string>
    </property>
   </widget>
   <widget class="QComboBox" name="source_lang">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="target_lang">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="swap_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
